app/controllers/auth_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+# AuthController exposes authorization endpoints for users to get access tokens
+class AuthController < ApplicationController
+  skip_before_action :verify_authenticity_token, only: :token
+  after_action :set_cors_header, only: :token
+
+  def authorize
+    params = request.params
+    if params[:client_id] == Rails.application.config.client_id
+      redirect_to "#{params[:redirect_uri]}?code=#{Rails.application.config.auth_code}&state=#{params[:state]}"
+    else
+      render json: { errors: ['Unauthorized client_id'] }, status: :unauthorized
+    end
+  end
+
+  def token
+    params = request.parameters
+    if params[:code] == Rails.application.config.auth_code
+      scope = ['launch/patient', 'patient/Immunization.read']
+      payload = { exp: helpers.convert_time_to_epoch(Time.current + 3600), scope: scope }
+      token = JWT.encode payload, Rails.application.config.hc_key.key, 'ES256'
```

Comment:
You should be able to use our existing HC code and not have to pull in JWT

Reply:
Oh wait this is a JWT and not a JWS... still with some refactoring we should be able use our existing code to take care of this without JWT. I'll leave that decision to @radamson though because it seems like this is a fairly experimental, testing-only PR.

Reply:
I was looking into utilizing the `JWS` class in the `HealthCards` library but it felt weird since creating a bearer token really has nothing to do with health cards. Up to you guys though I can go either way.

Reply:
I'm a little torn on this, but I do think it would be nice to use the code we already have just so we don't need to pull in another dependency and so we can eat our own dog food. Luckily, it's a pretty simple encode/decode and conveniently uses the same signing algorithm.

My vote is to use our `JWS` class in `health_cards`, but I could be swayed.

Reply:
Removed the dependency :)